quickstarts/router-ext/README.md

```diff
@@ -0,0 +1,253 @@
+
+```bash
+$ docker tag my-smartrouter-ext:1.0 default-route-openshift-image-registry.apps-crc.testing/rhpam-smartrouter/my-smartrouter-ext:1.0
+$ docker push default-route-openshift-image-registry.apps-crc.testing/rhpam-smartrouter/my-smartrouter-ext:1.0
+The push refers to repository [default-route-openshift-image-registry.apps-crc.testing/rhpam-smartrouter/my-smartrouter-ext]
+...
+1.0: digest: sha256:f3ab6429871c88815a8ebd3b0f4d38794a342980bb518f87a3d4512cd041f576 size: 1989
+```
+Note that the `rhpam-smartrouter` is the namespace we just created.
+
+Let's check if the image is there:
+
+```bash
+$ oc get imagestream
+NAME                 IMAGE REPOSITORY                                                                               TAGS      UPDATED
+my-smartrouter-ext   default-route-openshift-image-registry.apps-crc.testing/rhpam-smartrouter/my-smartrouter-ext   1.0       2 minutes ago
+```
+
+Now proceed to the OpenShift Web Console, use the following `crc` command to open it:
+```bash
+# opens the console in the web browser
+$ crc console
+# gets the crc credentials
+$ crc console --credentials
+```
+
+Once logged in, select the project we created before (rhpam-smartrouter) and install the Business Automation in this namespace.
+To do this please follow [this](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
+
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
+
+To configure the and Environment with Smart Router follow the steps described in this [section](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-smartrouter-proc).
+As we are going to use a custom Smart Router image, fill the following fields in the wizard installer:
+
+```bash
+Image Context -> rhpam-smartrouter (the namespace we've pushed the image previously)
+Image -> my-smartrouter-ext (the image name)
+Image tag -> 1.0
+```
+
+
+As output, a similar yaml should be generated:
+```yaml
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: smartrouter-ext-example
+spec:
+  environment: rhpam-trial
+  objects:
+    console:
+      replicas: 1
+    servers:
+      - id: kieserver
+        name: kieserver
+        replicas: 1
+    smartRouter:
+      imageContext: rhpam-smartrouter
+      image: my-smartrouter-ext
+      imageTag: '1.0'
+      replicas: 1
+
+```
+
+
+Make sure all the pods are running:
```

Comment:
```suggestion
Start the deployment. When the deployment completes, make sure all the pods are running:
```

Reply:
"Proceed by clicking on the `Deploy` button. When the deployment completes, make sure all the pods are running:"

wdyt?